_includes/banner.html

```diff
@@ -34,6 +34,7 @@
           <img
             class="usa-banner__icon usa-media-block__img"
             src="{{ site.baseurl }}/assets/img/icon-dot-gov.svg"
+            role="img"
```

Comment:
Can we discuss this further? You don't need an image role on this because you are using the correct img tag

Reply:
I'm just following @aduth 's suggestion https://github.com/18F/identity-site/pull/826#discussion_r880787900:

> Not sure why this is being removed. It was following canonical source for the design system component.
> 
> https://designsystem.digital.gov/components/banner/

Reply:
That is interesting. Perhaps because this is hidden only until the user drops the banner down?

I am still apprehensive of keeping this image tag in, but I might not understand how it plays out when the image starts out as hidden on page load. I would like to learn more about this

Reply:
I think it can go through. It would not impact functionality in any way, it would just be redundant.

Reply:
I think it's related to browser bugs, specifically this one: https://bugs.webkit.org/show_bug.cgi?id=145263

Reply:
Thanks for clarifying! Since this seems like it won't harm, but might fix a bug, I'll keep it and merge